tests/dummy/app/styles/app.scss

```diff
@@ -6,3 +6,45 @@
   max-height: 100vh;
   overflow: hidden;
 }
+
+.nav-sidebar {
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  top: 80px;
+  overflow-y: auto;
+  width: 240px;
+  border-right: 1px solid #dfdfdf;
+}
+
+.full-screen-content {
+  margin-left: 241px;
+}
+
+.full-screen-content > div {
+  height: calc(100vh - 80px);
+}
+
+.subnav-header {
+  margin: 10px;
+}
+
+.uk-navbar-container {
+  color: white;
+  background-color: #1e87f0 !important;
+}
+
+ul.uk-navbar-nav > li > a {
+  color: white;
+}
+
+ul.uk-navbar-nav > li.uk-active > a {
+  color: white;
+  font-weight: bolder;
+  font-size: larger;
+  margin-left: 20px;
+}
+
+.uk-navbar-nav {
+  margin-right: 20px;
+}
```

Comment:
Are those really needed? I don't see the class being used anywhere.

Reply:
it's used in the top navbar, it was giving it a better UI

Reply:
Ah, I see :+1: